### PR TITLE
DOC-4278

### PR DESCRIPTION
**Pull request by hghorbel**
PR for the 2nd iteration of Space Templates

### CODE REVIEW

File: docs/Manage-Space.rst [hunk 1]
```diff
@@ -227,8 +227,9 @@ follows:
 
 1. Click |image22| at the top left corner of the **Space** page to open the Add New Space form.
 
-2. In the Settings tab, enter a space name (required and from 3 to 200 characters) 
-   and description (from 0 to 255 characters).
+2. A popup is displayed containing all information related to the space creation.
```
Comment (plamarque) [inline docs/Manage-Space.rst:230]:
```suggestion
2. A popup is displayed containing all informations needed to create the space.
```

File: docs/Manage-Space.rst [hunk 2]
```diff
@@ -238,20 +239,22 @@ follows:
 
 .. _access-level-step:
 
-3. In the Access & Edit tab, select an access level.
+A pulldown lists all predefined templates that the space creator can use. A description of each template is displayed below the selectors.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
A pulldown lists all predefined templates that the space creator can use. A description of each template is displayed below.
```

File: docs/Manage-Space.rst [hunk 3]
```diff
@@ -238,20 +239,22 @@ follows:
...
+.. note:: Each template defines its default visibility, registration criteria and list of applications.
 
-|image31|
 
 .. _About-space-visibility:
 
--  **Visibility**: By default, the Visibility value is set to be Visible.
+4. The space creator may override the predefined visibility and registration options of the space.
+
+-  **Visibility**: By default, the Visibility value is set to be the one predefined in the template.
```
Comment (plamarque) [inline docs/Manage-Space.rst:256]:
```suggestion
-  **Hidden**: The visibility of the space is predefined by the template and can be overridden at creation time.
```

File: docs/Manage-Space.rst [hunk 4]
```diff
@@ -238,20 +239,22 @@ follows:
...
 
-	-  Visible: The space is always visible in the public spaces list.
+	-  No: The space is not hidden and always visible in the public spaces list.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
	-  No: The space is visible in the public spaces directory
```

File: docs/Manage-Space.rst [hunk 5]
```diff
@@ -238,20 +239,22 @@ follows:
...
 
-	-  Hidden: The space is not visible in the public spaces list.
+	-  Yes: The space is not visible in the public spaces list.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
	-  Yes: The space is hidden, hence it won't appear in the public spaces directory.
```

File: docs/Manage-Space.rst [hunk 6]
```diff
@@ -238,20 +239,22 @@ follows:
...
 
--  **Registration**: By default, the Registration value is set as
-   Validation.
+-  **Registration**: By default, the Registration value is set to be the one predefined in the template.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
-  **Registration**: The Registration policy is predefined by the template and can be overridden at creation time
```

File: docs/Manage-Space.rst [hunk 7]
```diff
@@ -261,35 +264,14 @@ follows:
...
-
-   -  **ii.** Select the group in the left pane, then its child group in
-      the right pane.
-
-      You will see your selected group as below.
-
-      |image25|
-
-.. note:: To remove the selected groups, untick the **Select a group of people to invite in your space** checkbox.
+5. In the **Users ** field, the space creator can invite users or space members to the space to create just by picking them with the selector.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
5. In the **Users ** field, the space creator can invite users or space members to the space just by picking them with the selector.
```

File: docs/Manage-Space.rst [hunk 8]
```diff
@@ -298,7 +280,7 @@ One of the two following cases occurs when you select this tab.
 **Sapce banner and avatar**
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
**Space banner and avatar**
```

File: docs/Manage-Space.rst [hunk 9]
```diff
@@ -261,35 +264,14 @@ follows:
...
-   After you have selected one existing group, all users in that group will be invited to join your newly created space.
+Type-ahead suggestions allow to facilitate selection users and groups to invite.
 
--  **The second instance**: You have not been a manager of any group yet. It means that this is the first time you create a space, so you cannot select a target group.
+.. note:: Only spaces that the creator is member of can be selected.
 
-   |image26|
 
-5. Click Create Space button to finish adding your new space. The new space appears.
+6. Click Create Space button to finish adding your new space. The new space appears.
```
Comment (plamarque) [inline docs/Manage-Space.rst]: @hghorbel  we should have some style  for representing product labels such as button names. @mzorai what do you suggest ?

File: docs/Manage-Space.rst [hunk 10]
```diff
@@ -298,7 +280,7 @@ One of the two following cases occurs when you select this tab.
 **Sapce banner and avatar**
 
 
-   By default, the space banner is grey (it is empty) and the space
+   By default, the space banner is set to be the one predefined in the template and the space
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
   By default, the space banner is predefined by the template and the space
```

File: docs/Manage-Space.rst [hunk 11]
```diff
@@ -298,7 +280,7 @@ One of the two following cases occurs when you select this tab.
...
    avatar contains the general space image:
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
   avatar contains the generic space image:
```

**Review by plamarque — CHANGES_REQUESTED** [review]
typos and language suggestions

File: docs/Manage-Space.rst [hunk 12]
```diff
@@ -261,35 +264,14 @@ follows:
...
-      |image25|
-
-.. note:: To remove the selected groups, untick the **Select a group of people to invite in your space** checkbox.
+5. In the **Users ** field, the space creator can invite users or space members to the space to create just by picking them with the selector.
 
-   After you have selected one existing group, all users in that group will be invited to join your newly created space.
+Type-ahead suggestions allow to facilitate selection users and groups to invite.
 
--  **The second instance**: You have not been a manager of any group yet. It means that this is the first time you create a space, so you cannot select a target group.
+.. note:: Only spaces that the creator is member of can be selected.
```
Comment (mzorai) [inline docs/Manage-Space.rst]:
```suggestion
.. note:: Only spaces where the creator is member could be selected.
```

**Review by mzorai — CHANGES_REQUESTED** [review]

File: docs/Manage-Space.rst (hunk 1 above)
Comment (mzorai) [reply docs/Manage-Space.rst:230]: @plamarque the word information is uncountable and should never take an "s" in the end

**Review by mzorai — CHANGES_REQUESTED** [review]

File: docs/Manage-Space.rst (hunk 9 above)
Comment (mzorai) [reply docs/Manage-Space.rst]: we can use these quotes ``Create Space``, they will appear in red

Comment (mzorai) [reply docs/Manage-Space.rst]: looks like the expression was interpreted. In fact we use the quotes `` around the word (with alt gr + 7 of the keyborad)

File: docs/Manage-Space.rst [hunk 13]
```diff
@@ -227,8 +227,9 @@ follows:
 
 1. Click |image22| at the top left corner of the **Space** page to open the Add New Space form.
 
-2. In the Settings tab, enter a space name (required and from 3 to 200 characters) 
-   and description (from 0 to 255 characters).
+2. A popup is displayed containing all informations needed to create the space.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
2. A popup is displayed containing all information needed to create the space.
```

as suggested by @mzorai  ;-)

**Review by plamarque — APPROVED** [review]

**Review by mzorai — APPROVED** [review]

**Pull request merged**